ui/widget.js

```diff
@@ -366,17 +368,39 @@ $.Widget.prototype = {
 
 		return this;
 	},
+	_updateClassesOption: function( value ) {
+		var classKey, elements, currentElements;
+		for ( classKey in value ) {
+			currentElements = this.classObject[ classKey ];
+			if ( value[ classKey ] === this.options.classes[ classKey ] ||
+					!currentElements ||
+					!currentElements.length ) {
+				continue;
+			}
+			elements = $( currentElements.get() );
+			this._removeClass( currentElements, classKey );
+			elements.addClass( this._classes({
```

Comment:
I feel like I've asked this before, but since its still not obvious: Why isn't this using `_addClass`?